static/sass/_pattern_stream.scss

```diff
@@ -12,6 +16,16 @@
     width: 100px;
   }
 
+  .p-stream__player {
+    position: absolute;
+    top: 0;
+
+    video {
```

Comment:
But I think it's not good practice to use HTML tags as selectors? And this seems to be only one level specificity.  If we are following BEM practices, this could lead to [poor selector-intent](https://cssguidelin.es/#selector-intent). what do you think @anthonydillon?

Reply:
The video tag is injected here and we don't really have control over it. I wouldn't usually use an element selector but I think that this is a fair case.

Reply:
oh... fair enough